lib/OidIndexGenerator.h

```diff
@@ -15,6 +15,7 @@ namespace sairedis
         public:
 
             virtual uint64_t increment() = 0;
+            virtual uint64_t incrementBy(uint64_t count) = 0;
```

Comment:
the idea here is that this function returns index, so this function should return list of new indexes

Reply:
also fix convention, each param in new line with In prefx

Reply:
Done